gallery/workbook/businesscontinuity-insights.json

```diff
@@ -35,7 +35,20 @@
 					"author": "Microsoft",
 					"description": "View information on user triggered operations on Recovery Services vaults like configuration changes and restores",
 					"name": "User Triggered Operations"
+				},
+				{
+					"id": "Workbooks/Azure Backup/ASR Job History",
+					"author": "Microsoft",
+					"description": "View information on ASR Job History",
+					"name": "ASR Job History"
+				},
+				{
+					"id": "Workbooks/Azure Backup/ASR ReplicatedItems ",
```

Comment:
"Workbooks/Azure Backup/ASR ReplicatedItems " ->"Workbooks/Azure Backup/ASR ReplicatedItems" (remove space at the end of the url)